### ISSUE TITLE
The impossible happened?

### ISSUE THREAD

**Issue by cinnamennen** [angr/angr#138]
I got `AngrCFGError: The impossible happened. Please report to Fish.`, so here goes. the binary is attached, and what was run was 
```python
import angr
project = angr.Project('../ctf/amadhj/amadhj')
cfg = project.analyses.CFGAccurate()
```
one thing to note is I have changed `path.py` to add `'_parent'` to the attributes of PathHistory (specifically in `__getstate__`)
I'd include the output, but it's a bit long, so I'll hold off on that unless it's requested.

[binary: amadhj.zip](https://github.com/angr/angr/files/307720/amadhj.zip)

**Comment by ltfish** [comment]
Can you check if you are using the latest version of angr from GitHub? We should have fixed this issue in commit 9b7ee3520b9ad3369994a1c9517b4ba05ab16cd6.

If you are using the version from PyPI, the bug is still there until we make the next release.

**Comment by cinnamennen** [comment]
I am using the PyPI version
